frontend/src/components/SearchBar.jsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react'
+import usePlacesAutocomplete, {getGeocode, getLatLng} from 'use-places-autocomplete'
+import {
+    Combobox,
+    ComboboxInput,
+    ComboboxPopover,
+    ComboboxList,
+    ComboboxOption,
+    ComboboxOptionText,
+  } from "@reach/combobox";
+import "@reach/combobox/styles.css";
+import { ToastContainer, toast } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
+
+const SearchBar = ({setPan, setTripArray}) => {
+    const {ready, value, setValue, suggestions: {status, data}, clearSuggestions} = usePlacesAutocomplete();
+    
+    const [currentAddress, setCurrentAddress] = React.useState(null);
+
+    const handleClick = () => {
+        if (currentAddress == null) {
+            toast.error("Please enter a destination!", {position: "top-center"});
+        }
+        else {
+            setTripArray(currentAddress);
+            toast.success("destination added!", {position: "top-center"});
+        }
+
+    }
+
+    const handleSelect = async (val) => {
+
+        // get value from select
+        setValue(val, false);
```

Comment:
Do this after the other stuff succeeds. Otherwise this just gets erased if there's an error